packages/headless/src/controllers/commerce/core/parameter-manager/headless-core-parameter-manager.ssr.ts

```diff
@@ -56,15 +56,23 @@ export function defineParameterManager<
           throw loadReducerError;
         }
         return buildProductListing(engine).parameterManager({
-          initialState: props.state ? props.state : props.initialState,
+          initialState: {
```

Comment:
This is a silly fix to work around the issue where the react providers are not passing the correct property for the initial state when rendering on the client.

We should not merge as-is, we don't want to add a useless prop on the controller.